python/cuml/manifold/umap.pyx

```diff
@@ -559,8 +617,19 @@ class UMAP(Base):
             New data to be transformed.
             Acceptable formats: cuDF DataFrame, NumPy ndarray, Numba device
             ndarray, cuda array interface compliant array like CuPy
+        knn_indices : arra-like (device or host) shape=(n_samples, n_neighbors)
+            knn_indices contains the indices of nearest neighbors for
```

Comment:
Same as above, and I would also state that the distance function used to compute the nearest neighbors should match the distance function that was used when training the embeddings. It's important to let users know this constraint because if they don't do this, they will get unexpected and potentially useless results.